kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
What does source.service mean?  It was intentional that we didn't have source.service since services are only targets of traffic, not sources of traffic.